test/download.test.js

```diff
@@ -30,6 +30,30 @@ describe("download", function () {
       });
   });
 
+  it("should download with custom tag", function (cb) {
+    var didSeeInfoPList = false;
+
+    download({
+      version: "22.3.6",
+      tag: "v22.3.6-20472245",
+      repo: "microsoft/vscode-gulp-electron",
```

Comment:
I'm wondering how the repo field works in this case? Should it be vscode-electron-prebuilt?

Reply:
It can be any repo that has the electron releases in the specific format uploaded, we are using this repo for testing purpose only.